app/src/test/java/com/mapzen/erasermap/TestMapController.java

```diff
@@ -0,0 +1,53 @@
+package com.mapzen.erasermap;
+
+import com.mapzen.tangram.LngLat;
+import com.mapzen.tangram.MapController;
+import com.mapzen.tangram.MapData;
+import com.mapzen.tangram.TestMapData;
+
+import android.opengl.GLSurfaceView;
+
+import java.util.HashSet;
+
+public class TestMapController extends MapController {
```

Comment:
This and `TestMapData` are nicely designed test doubles ;) but I don't currently see them used anywhere. Is there a plan to add these to certain tests later? Or maybe do we not need them after all?